test/blocking_test.c

```diff
@@ -155,6 +477,13 @@ int blocking_test()
     RUN_TEST(test_blocking_query);
     RUN_TEST(test_blocking_query_error);
     RUN_TEST(test_blocking_query_no_rows);
+    RUN_TEST(test_blocking_stmt_prepare);
+    RUN_TEST(test_blocking_stmt_execute_str);
+    RUN_TEST(test_blocking_stmt_execute_integer);
+    RUN_TEST(test_blocking_stmt_execute_double);
+    RUN_TEST(test_blocking_stmt_execute_datetime);
```

Comment:
Would it be possible to create one test for each type that's supported?

Reply:
Yeah that's the plan. I just haven't had a chance to sit down to do it yet 😅